charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -1,8 +1,24 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: pp-plugins-scheduler
+  name: aggregated-pp-plugins-scheduler-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
 rules:
+- apiGroups: ["*"]
+  resources: ["pods", "nodes"]
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:pp-plugins-scheduler
```

Comment:
Why do we need `system` prefix for the role here?

Reply:
I followed the `ClusterRole` in the [metrics-server](https://github.com/kubernetes-sigs/metrics-server/blob/99f1c4c8e2/charts/metrics-server/templates/clusterrole.yaml)